packages/backend-common/src/database/DatabaseManager.ts

```diff
@@ -352,7 +352,9 @@ export class DatabaseManager {
     let lastKeepaliveFailed = false;
 
     setInterval(() => {
-      client.raw('select 1').then(
+      // During testing it can happen that the environment is torn down and
+      // this client is `undefined`, but this interval is still run.
+      client?.raw('select 1').then(
```

Comment:
What. How can it change its value?

Reply:
![image](https://user-images.githubusercontent.com/4984472/188967398-0f222dee-09b3-4564-b9b5-a5874567d066.png)

![image](https://user-images.githubusercontent.com/4984472/188968057-a7d9e3bf-6500-408b-abf2-a5ed47e6d257.png)

I have no idea 😅

Reply:
Computers were a mistake.